.evergreen-functions.yml

```diff
@@ -540,7 +540,7 @@ functions:
         shell: bash
         <<: *e2e_include_expansions_in_env
         working_dir: src/github.com/mongodb/mongodb-kubernetes
-        binary: scripts/dev/run_python.sh scripts/release/pipeline_main.py --parallel ${image_name}
+        binary: scripts/dev/run_python.sh scripts/release/pipeline_main.py --parallel ${image_name} ${all_agents}
```

Comment:
all_agents expansion is empty, but in the manual release agents on ecr variant it will be set to `--all-agents`